src/lib.rs

```diff
@@ -452,12 +468,21 @@ impl Client {
             }
         };
 
+        let mut default_tags = options.default_tags;
+        // Prepending default tags in order to let the user be able to override system tags with custom
+        // default tags
```

Comment:
I think this should be 
```suggestion
        // Append default_system_tags default tags in order to let the user override system tags
```